bal/wallet.py

```diff
@@ -29,24 +33,28 @@ def init_wallet(keystore_path):
     try:
         with open(PRIVATE_KEY_LOCATION, "w") as f:
             f.write(new_private_key)
-    except:
+    except BaseException:
         return
     print('new wallet with private key created to : %s', PRIVATE_KEY_LOCATION)
 
+
 def delete_wallet():
     if os.path.exists(PRIVATE_KEY_LOCATION):
-      os.remove(PRIVATE_KEY_LOCATION)
+        os.remove(PRIVATE_KEY_LOCATION)
     else:
-      print("The file does not exist")
+        print("The file does not exist")
```

Comment:
why one more indentation?

Reply:
This tab size did not match with the size of other tabs in the file.